apps/ledger-live-desktop/tools/config/swapConnectWebviewPreloader.esbuild.js

```diff
@@ -1,12 +0,0 @@
-const common = require("./common.esbuild");
-const { electronPreloaderExternals } = require("esbuild-utils");
-
-module.exports = {
-  ...common,
-  entryPoints: ["./src/swapConnectWebviewPreloader/index.ts"],
-  entryNames: "swapConnectWebviewPreloader.bundle",
-  platform: "node",
-  format: "cjs",
-  target: ["chrome91"],
-  external: [...electronPreloaderExternals],
-};
```

Comment:
This is the mysterious stuff that FTX used to hook into Ledger Live. For example when the user completes their auth on the FTX webview, FTX would send a `setToken` message to the webview host (Ledger Live in this case), and this swapConnectWebview stuff started the listeners that were listening for the `setToken` message which would allow us to set the auth token in the user's local storage. So we can remove all this stuff